packages/core/src/components/icon/_icon.scss

```diff
@@ -3,21 +3,28 @@
 
 @import "~@blueprintjs/icons/src/icons";
 
-// the SVG icon class
+// the icon class which will contain an SVG icon
 .#{$ns}-icon {
-  // remove any extra inline space and ensure centering with text
-  display: inline-flex;
+  // ensure icons sit inline with text & isolate svg from surrounding elements
+  // (vertical alignment in flow is usually off due to svg - not an issue with flex.)
```

Comment:
would it be a good idea to refer to the github issue here for context?

Reply:
there's no issue, it's just a thing.